decidim-blogs/lib/decidim/api/post_type.rb

```diff
@@ -17,6 +18,11 @@ class PostType < Decidim::Api::Types::BaseObject
       field :id, GraphQL::Types::ID, "The internal ID of this post", null: false
       field :published_at, Decidim::Core::DateTimeType, "The time this page was published", null: false
       field :title, Decidim::Core::TranslatedFieldType, "The title for this post", null: true
+      field :url, String, "The URL for this post", null: false
```

Comment:
Let's add the new field to `decidim-blogs/spec/types/integration_schema_spec.rb`

Reply:
I am assuming you already reviewed everything from this PR, so i have implement it in #14358 , commit: 0609b872aca75ba4de3a97eb0834b4cbcc58dd48